src/embed/IVbrickApi.ts

```diff
@@ -3,98 +3,44 @@ import { PlayerStatus } from './PlayerStatus';
 export { WebcastStatus } from './WebcastStatus';
 export { PlayerStatus } from './PlayerStatus';
 
-export interface IVbrickAPIToken {
-	type: TokenType;
 
-	/**
-	 * String containing the token value
-	 */
-	value: string;
-
-	/**
-	 * The issuer for the token
-	 */
-	issuer: string;
-}
-
-export enum TokenType {
-	JWT = 'JWT',
-	ACCESS_TOKEN = 'AccessToken'
-}
-
-export interface IVbrickEmbedConfig {
-	token: IVbrickAPIToken;
-
-	/**
-	 * URL for Rev
-	 */
-	baseUrl: string;
-
-	/**
-	 * An optional class to be set on the iframe.
-	 */
-	className: string;
-
-	/**
-	 * Width of the iframe. Default is "100%"
-	 */
-	width: string;
+export interface IVbrickVideoEmbed extends IVbrickBaseEmbed {
 
-	/**
-	 * Height of the iframe. Default is "100%"
+		/**
+	 * video playing, buffering, etc
 	 */
-	height: string;
+	readonly playerStatus: PlayerStatus;
 
-	/**
-	 * If true, enable logging messages in the browser console.
+	 /**
+	 * Player Volume. 0-1
 	 */
-	log?: boolean
-}
+	readonly volume: number;
 
-export interface IVbrickWebcastConfig extends IVbrickEmbedConfig {
 	/**
-	 * Optional, default is true.
+	 * Whether captions are enabled, and selected language
 	 */
-	showVideo?: boolean;
-}
+	readonly captions: ICaptionSettings;
```

Comment:
`captions` is only on `IVbrickVideoEmbed`, but the `captionsChanged` event is associated with the parent interface `IVbrickBaseEmbed`. Should `captions` apply to Webcast as well? or just on base? If it's future functionality then only apply on relevant interface for now

Reply:
Thats how the stories were written. 
The player state will be moved in a future story